manuscript/simulators.tex

```diff
@@ -0,0 +1,27 @@
+\section{Fuel Cycle Simulators}
+
+\subsection{\gls{VISION}}
+\gls{VISION} is a fuel cycle simulation
+that is intended to evaluate "what if" scenarios.
+It considers the nuclear fuel cycle at a systems level,
+not in individual facilities \cite{jacobson_user_2011}.
```

Comment:
correct. We call it a 'fleet based' simulator rather than a 'discrete facility' simulator.